### DISCUSSION TITLE
nova2-medical for languages other than English (Dutch ,  Spanish, French)

### DISCUSSION THREAD

**Discussion by chicodespons** [deepgram/community#689] [Feature Requests]
Hello,
 While I noted the release of Nova-2 with extended language support for Spanish, Hindi, German, and more, I am particularly interested in the availability of medical speech-to-text transcription services for Dutch and French languages. Could you provide any updates or insights regarding these languages?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@chicodespons  Thanks for letting us know about your languages needs for Medical audio. Currently I don't have specifics of when we'll be releasing more languages for our medical models but I'll pass this onto our Product Team.

**Reply by chicodespons** [reply]
@jpvajda Thanks for taking the time to answer my question. I am making a medical registration tool and would like to know if deepgram can meet my needs!

**Reply by chicodespons** [reply]
@jpvajda Do you have more information concerning my question?

**Reply by jpvajda** [reply]
@chicodespons  I've passed this feedback along to our product team but there is currently no timeline I'm aware of for expanding nova-2-medical language support.

 If you want to join our [Discord Community](https://dpgr.am/discord) we constantly post updates to product releases and it's a good way to keep uo on the new models we are releasing.

**Comment by rmasad** [comment]
Interested in nova2-medical in Spanish
